lib/components/Modal/Bottom/modal-image.tsx

```diff
@@ -0,0 +1,32 @@
+import clsx from "clsx";
+
+interface ModalBodyProps {
+    className?: string;
+    src?: string;
+    height?: string;
+}
+
+export const ModalImage = ({
+    children,
+    className,
+    src,
+    height = "var(--temp-static-spacing-210)",
+}: React.PropsWithChildren<ModalBodyProps>) => (
+    <>
+        {src ? (
```

Comment:
I want to give an opportunity to this component to be flexible and to accept ReactNode with img OR to accept just the url, which will be used as a background-image.